docs/_source/conf.py

```diff
@@ -199,3 +200,5 @@
     ('py:class', 'json.encoder.JSONEncoder'),
     ('py:class', 'sceptre.config.reader.Attributes')
 ]
+
+set_type_checking_flag = True
```

Comment:
This allows Sphinx to "understand" type annotations where the import is declared in an if TYPE_CHECKING condition.